package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vavato-ui",
-  "version": "1.4.15",
```

Comment:
Now, this will be updated by default, in a GitHub Action.

Reply:
Awesome, thanks.